[2025-04-10T19:06:24.792+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-10T19:06:24.815+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: approval_workflow.fetch_data manual__2025-04-10T19:06:22.638774+00:00 [queued]>
[2025-04-10T19:06:24.830+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: approval_workflow.fetch_data manual__2025-04-10T19:06:22.638774+00:00 [queued]>
[2025-04-10T19:06:24.830+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-10T19:06:24.853+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-04-10 19:06:22.638774+00:00
[2025-04-10T19:06:24.862+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=143) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-10T19:06:24.864+0000] {standard_task_runner.py:72} INFO - Started process 145 to run task
[2025-04-10T19:06:24.865+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'approval_workflow', 'fetch_data', 'manual__2025-04-10T19:06:22.638774+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/custom_dag.py', '--cfg-path', '/tmp/tmpenqlkdai']
[2025-04-10T19:06:24.869+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask fetch_data
[2025-04-10T19:06:24.999+0000] {task_command.py:467} INFO - Running <TaskInstance: approval_workflow.fetch_data manual__2025-04-10T19:06:22.638774+00:00 [running]> on host b99a7f546fb2
[2025-04-10T19:06:25.386+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='approval_workflow' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T19:06:22.638774+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-10T19:06:22.638774+00:00'
[2025-04-10T19:06:25.390+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-10T19:06:25.452+0000] {logging_mixin.py:190} INFO - fetch_data execute..
[2025-04-10T19:06:25.453+0000] {logging_mixin.py:190} INFO - Files in directory: ['data.xlsx']
[2025-04-10T19:06:25.453+0000] {logging_mixin.py:190} INFO - data.xlsx
[2025-04-10T19:06:25.669+0000] {logging_mixin.py:190} INFO -    ID     Name  Age Department     Contact            Email
0   1    Alice   30         HR  12345678.0  test1@gmail.com
1   2      Bob   25         IT  98765432.0  test2@gmail.com
2   3  Charlie   35    Finance         NaN  test3@gmail.com
[2025-04-10T19:06:25.703+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-10T19:06:25.716+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-10T19:06:25.717+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=approval_workflow, task_id=fetch_data, run_id=manual__2025-04-10T19:06:22.638774+00:00, execution_date=20250410T190622, start_date=20250410T190624, end_date=20250410T190625
[2025-04-10T19:06:25.804+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-10T19:06:25.859+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-10T19:06:25.870+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
